manager/assets/modext/widgets/core/modx.searchbar.js

```diff
@@ -97,25 +110,35 @@ MODx.SearchBar = function(config) {
                     this.tpl.type = null;
                 }
             }
-            ,focus: this.focusBar
-            ,blur: this.blurBar
+            ,focus: {
+                fn: function(){
+                    this.focusBar();
+                }
+            }
+            ,blur: {
+                fn: function(){
+                    this.blurBar()
+                }
+            }
             ,scope: this
         }
+
     });
+
     MODx.SearchBar.superclass.constructor.call(this, config);
-    //this.setKeyMap();
+    this.setKeyMap();
```

Comment:
Agreed. Enabling the keymap (to allow ESC closing as you say), but remove the conflicting shortcut would cover both concerns.